examples/scala/README.md

```diff
@@ -0,0 +1,17 @@
+# Scala 3
+
+*Insert snarky language description here.*
```

Comment:
:-) Kan ju vara ganska enkelt och osnarky också, t.o.m. kärleksfullt. Hur är det med inspirationen? Jag borde inte köra denna för då blir det typ: "Scala is pretty much a dead language, even Lightbend and Martin Odersky don't talk about it anymore, and people who have been burned by its complexity have long since migrated to simpler alternatives like Go, or even back to Java. Some suckers for punishment have moved to Haskell instead."

Reply:
Äsch, jag tar din beskrivning rakt av. Vi får göra avbön om vi har några Scala-fundamentalister på bygget, men jag känner nog ingen sån, såvitt jag vet. Uppdatering om två minuter.